servicemesh-install/topics/appendix-threescale-adapter.adoc

```diff
@@ -21,10 +22,35 @@ You can use these examples to configure requests to your services using the 3sca
 Pay particular attention to the `kind: handler` resource. You must update this with your 3scale credentials and the service ID of the API you want to manage.
 ====
 
-Modify the handler configuration with your 3scale configuration.
+. Modify the handler configuration with your 3scale configuration.
++
+[source,yaml]
+----
+  apiVersion: "config.istio.io/v1alpha2"
+  kind: handler
+  metadata:
+   name: threescale
+   namespace:
+  spec:
+   adapter: threescale
+   params:
+     service_id: "<SERVICE_ID>"
+     system_url: "https://<organization>-admin.3scale.net/"
+     access_token: "<ACCESS_TOKEN>"
+   connection:
+     address: "threescale-istio-adapter:3333"
+----
+
++
+. Create the required resources:
++
+----
+oc create -f istio/ -n
+----
+
 
 == Generating custom resources
-The adapter embeds a tool that allows generation of the `handler`, `instance`, and `rule` custom resources.
+The adapter includes a tool that allows you to generate the `handler`, `instance`, and `rule` custom resources.
```

Comment:
@dfennessy @PhilipGough @JStickler I changed the language here to clarify. Does this help?

Reply:
LGTM

Reply:
+1